js/src/personal/kazi/Kazi.page.tsx

```diff
@@ -1,4 +1,4 @@
-import { Title, Text, useMantineColorScheme } from '@mantine/core'
+import { Button, Title, Text, useMantineColorScheme } from '@mantine/core'
```

Comment:
Your commits should likely be squashed together. I review the code commit by commit, so if you have things that get overwritten in there, they should likely be squashed.

Reply:
Added new component and tested manually.